SimCalorimetry/EcalSimAlgos/src/EcalLiteDTUCoder.cc

```diff
@@ -0,0 +1,241 @@
+#include "SimCalorimetry/EcalSimAlgos/interface/EcalLiteDTUCoder.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "SimGeneral/NoiseGenerators/interface/CorrelatedNoisifier.h"
+#include "DataFormats/EcalDigi/interface/EcalLiteDTUSample.h"
+#include "DataFormats/EcalDigi/interface/EcalDataFrame_Ph2.h"
+
+#include "CondFormats/EcalObjects/interface/EcalConstants.h"
+
+#include <iostream>
+
+//#include "Geometry/CaloGeometry/interface/CaloGenericDetId.h"
+
+/////////////////////
+//NOTE:
+//TO BE CORRECTLY IMPLEMENTED!
```

Comment:
does this imply it's not correctly implemented now?

Reply:
comment removed